collector/src/main/java/com/bitmovin/analytics/utils/DataSerializer.kt

```diff
@@ -0,0 +1,27 @@
+package com.bitmovin.analytics.utils
+
+import com.google.gson.Gson
+import com.google.gson.JsonSyntaxException
+
+object DataSerializer {
+    @JvmStatic
+    fun <T> serialize(data: T?): String? {
```

Comment:
```suggestion
fun <T> serialize(data: T?): String? {
        data ?: return null
        return Gson().toJson(data)
    }
```

Reply:
The `@JvmStatic` annotations are necessary for the java classes that access those methods